src/System.Drawing.Common/src/System/Drawing/Bitmap.Windows.cs

```diff
@@ -2,12 +2,9 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
-using System.ComponentModel;
-using System.Drawing.Imaging;
 using System.Drawing.Internal;
 using System.IO;
-using System.Runtime.InteropServices;
-using System.Security.Permissions;
+using Gdip = System.Drawing.SafeNativeMethods.Gdip;
```

Comment:
Has there been any consideration towards just doing `using static System.Drawing.SafeNativeMethods.Gdip`?

That would allow code like:
```C#
int status = Gdip.GdipCreateBitmapFromStream(new GPStream(stream), out bitmap);
Gdip.CheckStatus(status);
```

To just be:
```C#
int status = GdipCreateBitmapFromStream(new GPStream(stream), out bitmap);
CheckStatus(status);
```

Reply:
https://github.com/dotnet/corefx/issues/6408